pages/create.js

```diff
@@ -178,7 +178,14 @@ class CreateTournamentForm extends React.Component {
     }
 
     handleGroupAdvanceInput(input) {
-        this.setState({groupAdvance: input.target.value});
+        const newAdvance = input.target.value;
+
+        if (newAdvance === undefined || newAdvance < 0 ||
```

Comment:
this could be `newAdvance <= 0`

Reply:
Agreed, will change and force push, if you want me to